CFC_WebApp/tests/TestCarbon.py

```diff
@@ -249,6 +249,13 @@ def testGetCarbonFootprintsForMap(self):
     footprintMap = carbon.getCarbonFootprintsForMap(testDistanceMap, testModeFootprintMap)
     self.assertEqual(footprintMap, {'a': 0.001, 'b': 0.004, 'c': 0.009})
 
+  def testGetCarbonFootprintsForMap(self):
+    testDistanceMap = {'a': 1, 'b': 2, 'c': 3}
+    testModeFootprintMap = {'a': 1, 'b': 2, 'c': 3}
+
+    footprintMap = carbon.getCarbonFootprintsForMap(testDistanceMap, testModeFootprintMap)
```

Comment:
Isn't this the same as the previous test? Shouldn't you be making a call to user instead?